src/yes/yes.rs

```diff
@@ -51,7 +53,13 @@ pub fn uumain(args: Vec<String>) -> i32 {
         matches.free.join(" ")
     };
 
-    exec(&string[..]);
+    let mut multistring = string.clone();
+    while multistring.len() < BUF_SIZE - string.len() - 1 {
+        multistring.push_str("\n");
+        multistring.push_str(&string);
+    }
+
+    exec(&multistring[..]);
```

Comment:
You can just change all the added stuff here to `exec(&string.repeat(BUFSIZE / string.len()));`